docs/architecture/introduction.rst

```diff
@@ -39,8 +39,9 @@ write code conforming to the trial APIs. Commands are useful for running existin
 manner; shells provide access to the cluster in the form of interactive `SSH
 <https://en.wikipedia.org/wiki/SSH_(Secure_Shell)>`_ sessions.
 
-This document provides an overview of the most common CLI commands related to shells and commands;
-see :ref:`cli` for full documentation.
+.. note::
+
+   For information about the Determined CLI, visit :ref:`cli-ug`.
```

Comment:
While it's true that you use commands and shells through the CLI, you also use the CLI for many of the other sections here (Queue Management, Notebooks, Tensorboards, Experiments, etc).  So maybe it's worth having a small CLI section which says basically that? That there exists a powerful CLI tool which represents one of the main entrypoints to the Determined system?

Reply:
resolved